xarray/tests/test_plot.py

```diff
@@ -975,13 +972,9 @@ def test_cmap_and_color_both(self):
 
     def test_2d_coord_with_interval(self):
         for dim in self.darray.dims:
```

Comment:
I left the loop here, because for the 2d plots, x and y axis are treated separately.